core/includes/theme.inc

```diff
@@ -2022,10 +2022,7 @@ function theme_breadcrumb($variables) {
   $breadcrumb = $variables['breadcrumb'];
   $output = '';
   if (!empty($breadcrumb)) {
-    $output .= '<nav class="breadcrumb">';
-    // Provide a navigational heading to give context for breadcrumb links to
-    // screen-reader users. Make the heading invisible with .element-invisible.
-    $output .= '<h2 class="element-invisible">' . t('You are here') . '</h2>';
+    $output .= '<nav class="breadcrumb" aria-label="Website Orientation">';
```

Comment:
"Website Orientation" should be wrapped in t() - otherwise it's not translatable. Same in both theme templates.

Reply:
@indigoxela  Should I wrap the aria label text in t()? Or should I separately put Website Orientation outside of the nav element?

Reply:
@developer-diganta - I didn't know the answer, but I basically just did a search for `aria-label` in the code base to see how it is done elsewhere. There are lots of examples. Based upon those examples. This one looks particularly helpful:

![image](https://user-images.githubusercontent.com/3144571/167989852-0db867fc-1021-4487-89d6-6da6e03fb6f3.png)